lib/optimizely/version.rb

```diff
@@ -15,4 +15,5 @@
 #
 module Optimizely
   VERSION = '1.3.0'.freeze
+  CLIENT_ENGINE = 'ruby-sdk'.freeze
```

Comment:
What's the reason behind it?

Reply:
ruby-sdk is used as string from multiple places in previous event_builder code which was updated to use new endpoint. Since this is a constant i have freeze it like version

https://github.com/optimizely/node-sdk/blob/94f940cff1872ec1023fd9eb2ad4a3781d31d61d/lib/utils/enums/index.js#L106